### PR TITLE
updating deprecated string in config

### PR DESCRIPTION
**Pull request by mabicca**
https://github.com/Azure/azure-linux-extensions/issues/242

~ has been deprecated and we should use "stop"

More info at:
http://www.rsyslog.com/doc/v8-stable/compatibility/v7compatibility.html

"omruleset and discard (~) action are deprecated
Both continue to work, but have been replaced by better alternatives.

The discard action (tilde character) has been replaced by the “stop” RainerScript directive. It is considered more intuitive and offers slightly better performance.

The omruleset module has been replaced by the “call” RainerScript directive. Call permits to execute a ruleset like a subroutine, and does so with much higher performance than omruleset did. Note that omruleset could be run off an async queue. This was more a side than a desired effect and is not supported by the call statement. If that effect was needed, it can simply be simulated by running the called rulesets actions asynchronously (what in any case is the right way to handle this).

Note that the deprecated modules emit warning messages when being used. They tell that the construct is deprecated and which statement is to be used as replacement. This does not affect operations: both modules are still fully operational and will not be removed in the v7 timeframe."

### CODE REVIEW

File: Diagnostic/rsyslog5/omazurelinuxmds.conf [hunk 1]
```diff
@@ -25,4 +25,4 @@ $ActionQueueSaveOnShutdown on
 $ActionQueueFileName lad_mdsd_queue
 $ActionQueueDiscardSeverity 8
 local6.* :omazuremds:;fmtfile
-& ~
+& stop
```
Comment (hosungsmsft) [inline Diagnostic/rsyslog5/omazurelinuxmds.conf]: @mabicca : Could you please double-check if this is supported on rsyslog v.5? If not, we shouldn't change this here.

Comment (mabicca) [inline Diagnostic/rsyslog5/omazurelinuxmds.conf]: Hi @hosungsmsft yes, sorry, somehow that didn't get through properly, yes, the change should only be made for rsyslog7 and later. is there a quick way for you to fix that or you want me to do that?

Comment (hosungsmsft) [inline Diagnostic/rsyslog5/omazurelinuxmds.conf]: Thanks for confirming @mabicca . Please allow me to apply this change only to ver. 7 & 8, to reduce your inconvenience. We are planning to release a newer version of LAD soon, so this change should be included there for sure, together with the queue file name change.

Comment (mabicca) [inline Diagnostic/rsyslog5/omazurelinuxmds.conf]: Hi @hosungsmsft alright, yes, I wanted to see exactly what those config sections mean too , but I would probably change the filenames just to be safe ;) thanks for the updates.

**Comment by hosungsmsft** [comment]
This change is applied to lad-2.3.9009 branch, which is now being merged to master, so closing this PR without taking it. Thanks.

**Pull request closed without merging**